src/sherpa_ai/tools.py

```diff
@@ -209,3 +218,31 @@ def _run(self, query: str) -> str:
 
     def _arun(self, query: str) -> str:
         raise NotImplementedError("UserInputTool does not support async run")
+
+class HugChatTool(BaseTool):
+    name = "Hugchat"
+    description = (
+        "Access the user input for the task."
+        "This tool is an alternative way to to ask clarifying questions to solve the task, using HuggingChat via the HugChat API."
+    )
+    def _run(self, query: str) -> str:
+
+        # Log in to huggingface and grant authorization to huggingchat
+        sign = Login(cfg.HUGCHAT_EMAIL, cfg.HUGCHAT_PASS)
+        # Save cookies to the local directory
+        cookies = sign.login()
+
+        # Save cookies to the local directory
+        cookie_path_dir = "./cookies_snapshot"
```

Comment:
I think even if we cannot avoid the cookies, we don't need to save them locally. We can remove line 236 and 237